packages/desktop-client/src/components/accounts/MobileAccount.js

```diff
@@ -88,12 +89,9 @@ function Account(props) {
     [dispatch],
   );
 
-  const { id: accountId } = props.match.params;
+  const { id: accountId } = useParams();
 
```

Comment:
The `{}` is only used in destructuring

```suggestion
  const makeRootQuery = () => queries.makeTransactionsQuery(accountId);
```

Reply:
Not sure I follow: can't `id` be falsey on line 92, which could break `makeTransactionsQuery`?

Was the existing fallback `{}` unnecessary?